templates/partial/_navigation.html

```diff
@@ -1,39 +1,44 @@
-<header id="navigation" class="p-navigation">
-  <div class="p-navigation__row">
-    <div class="p-navigation__banner">
-      <div class="p-navigation__logo u-vertically-center">
-        <a class="p-navigation__item" href="/" style="color:white;">
-          MicroK8s
+<a href="#main-content" class="p-link--skip">Jump to main content</a>
+
+<header id="navigation" class="p-navigation is-dark">
+  {% block header_banner %}
+  <div class="row">
+    <div class="p-navigation__banner col-3">
+      <div class="p-navigation__tagged-logo">
+        <a class="p-navigation__link" href="/">
+          <div class="p-navigation__logo-tag">
+            <img class="p-navigation__logo-icon" src="https://assets.ubuntu.com/v1/82818827-CoF_white.svg" alt="Canonical" width="95">
+          </div>
+          <span class="p-navigation__logo-title">MicroK8s</span>
         </a>
       </div>
       <a href="#navigation" class="p-navigation__toggle--open" title="menu">Menu</a>
       <a href="#navigation-closed" class="p-navigation__toggle--close" title="close menu">Close menu</a>
     </div>
 
-    <nav class="p-navigation__nav" aria-label="Main">
-      <ul class="p-navigation__items">
-        <li class="p-navigation__item {% if request.path in ['/high-availability',] %}is-selected{% endif %}">
-          <a class="p-navigation__link" href="/high-availability">Features</a>
+    <nav class="p-navigation__nav col-9" aria-label="main-navigation">
+      <ul class="p-navigation__items" style="margin-left: -.75rem;">
+        <li class="p-navigation__item">
```

Comment:
Looks like these links are missing the `is-selected` if the `request.path` matches.